src/api/messaging.js

```diff
@@ -0,0 +1,73 @@
+import { handleResponse, handleError } from './apiUtils';
+import { session } from '../models/auth';
+
+export default {
+  getRegion() {
+    const query = `https://dev-k8s.treetracker.org/messaging/region`;
```

Comment:
This URL should not be hard-coded. Maybe we can load the URL using an environment variable?